teuthology/worker.py

```diff
@@ -222,7 +222,7 @@ def run_with_watchdog(process, job_config):
         report.try_push_job_info(job_info, dict(status='dead'))
 
 
-def run_job(job_config, teuth_bin_path):
+def run_job(job_config, teuth_bin_path, verbose):
```

Comment:
If you don't provide a default value for the verbose arg, this will break:
https://github.com/dachary/teuthology/blob/wip-worker-verbose/teuthology/worker.py#L165